src/App.tsx

```diff
@@ -1,10 +1,93 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { TodosList } from './components/TodoList/TodoList';
+import { TodoHeader } from './components/TodoHeader/TodoHeader';
+import { TodoFooter } from './components/TodoFooter/TodoFooter';
+import { TodoError } from './components/TodoError/TodoError';
+
+const statusTodos = (todos: Todo[], status: string) => {
+  let filteredTodos = todos;
+
+  if (status === 'active') {
+    filteredTodos = todos.filter(item => !item.completed);
+  } else if (status === 'completed') {
+    filteredTodos = todos.filter(item => item.completed);
+  } else {
+    filteredTodos = todos;
+  }
+
+  return filteredTodos;
+};
 
-const USER_ID = 0;
+const USER_ID = 6709;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isActive, setIsActive] = useState(false);
+  const [activeFilter, setActiveFilter] = useState('all');
+  const [error, setError] = useState('');
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then((res) => {
+        setTodos(res);
+      })
+      .catch(() => {
+        setError('Unable to add a todo');
+        setTimeout(() => {
+          setError('');
+        }, 3000);
```

Comment:
You should clear all timers on the component unmount to prevent unpredictable side effects https://bobbyhadz.com/blog/react-cleartimeout

Reply:
@Vasyliy22 
Try to make the next approach.
In useEffect hook you can return function which will clear timeout (you could declare const with future timeout as the first variable inside useEffect).